Proposal.md

```diff
@@ -83,23 +83,22 @@ class parser
 }
 ```
 
-It’s a very better solution than the previous one because code complexity is not increased by `if` statement, and code readability is improved.
-But now imagine that you use it in lot of classes: you must define lot of `blackhole*` classes, and doing that can be rapidly a PITA!
-So, i think that it must be interesting to implements natively in the language the null object pattern as a class which has the following behavior:
-
-1) an instance can replace any interface or class in term of type hinting (so the class implements or extends virtually all classes and interfaces);
-2) all call to a method upon an instance return `$this` to avoiding the special case of returning "no object ».
-3) an instance do absolutely nothing!
+It’s a much better solution than the previous one because code complexity is not increased by this `if` statement, and code readability is improved.
+But now imagine you use it in lots of classes: you must define lots of `blackhole*` classes, and doing that can be rapidly a PITA!
+So, we think it would be interesting to natively implement the Null Object Pattern in PHP as a class which would have the following behaviors:
 
-Its name can be:
+1. an instance can replace any interface or class in term of type hinting (so the class virtually implements or extends all classes and interfaces)
+2. every call to a method upon an instance return `$this` to avoid the special case of returning "no object".
+3. an instance does absolutely nothing!
 
+Its name could be:
```

Comment:
J'aurai pas mis le Its même si grammaticalement correct, mais plutôt un truc du genre You can name it, etc..